juriscraper/opinions/united_states/state/wisctapp.py

```diff
@@ -0,0 +1,46 @@
+from lxml import etree
+
+from juriscraper.opinions.united_states.state import wis
+
+
+class Site(wis.Site):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.court_id = self.__module__
+        self.base_url = "https://www.wicourts.gov/other/appeals/caopin.jsp"
+        self.set_url()
+        # # 2024 WI App 36
+        self.cite_regex = (
+            r"(?P<volume>20\d{2})\s(?P<reporter>WI App)\s(?P<page>\d+)"
+        )
+
+    @staticmethod
+    def set_status(caption: etree._Element) -> str:
+        """Set status of opinions
+
+        Published opinions are either identified as marked for publication
+        or if they are older opinions with Final publication date ... etc
+
+        :param caption: The case name field
+        :return:Status
+        """
+        notes = caption.xpath("./strong/text()")
+        if notes and "publication" in notes[0]:
+            status = "Published"
+        else:
+            status = "Unpublished"
+        return status
```

Comment:
I think there is an extra status, Errata. The contents of the documents
![image](https://github.com/user-attachments/assets/c0be7ab0-719c-4525-a464-d1f5f5c8fb0f)

How they look on the results lists:
![image](https://github.com/user-attachments/assets/b45c333e-3688-4694-869a-3be252449a8f)

Reply:
thanks - I think ive accounted for errata status now

Reply:
This works to properly tag Errata documents, but it has false positives, for files that have the disclaimer that an Errata was published at a later date
For example, these contain opinions, and have an errata disclaimer:

![image](https://github.com/user-attachments/assets/5b4eb5dd-4e97-443f-bc9f-d92c7dab4dab)

These are the Errata
![image](https://github.com/user-attachments/assets/9d2afa7c-dc2f-418c-873b-55d6b83a3d52)

Both sets have status Errata, and they shouldn't